packages/paste-core/utilities/grid/src/helpers.tsx

```diff
@@ -0,0 +1,71 @@
+import {ThemeShape} from '@twilio-paste/theme-tokens';
+import {ResponsiveValue} from 'styled-system';
+import {Margin, Space, SpaceOptions} from '@twilio-paste/style-props';
+import {Vertical} from '@twilio-paste/flex';
+import {
+  ColumnOffset,
+  ColumnOffsetOptions,
+  ColumnProps,
+  ColumnSpanOptions,
+  ColumnMinWidth,
+  ColumnWidthSpan,
+} from './types';
+
+export const getNegativeMargin = (theme: ThemeShape, gutter?: Space): Margin => {
+  if (Array.isArray(gutter)) {
+    return (gutter as SpaceOptions[]).map((value: SpaceOptions) => {
+      return `-${theme.space[value]}` as SpaceOptions;
+    });
+  }
+
+  if (gutter) {
+    return `-${theme.space[gutter as SpaceOptions]}` as SpaceOptions;
+  }
+
+  return 'auto';
+};
+
+export const getFlexDirection = (vertical: Vertical): ColumnMinWidth => {
```

Comment:
Even though this is exported from another file in the package, when it's part of the core bundle would another exported method called `getFlexDirection` from another package, when bundled together still cause a name  clash @TheSisb

Reply:
This is safe because when `core` imports packages it does it like this:

![image](https://user-images.githubusercontent.com/1592327/73694542-de344a00-469d-11ea-816f-e8515200cbf0.png)

When we import from a package name rather than from a file (i.e.: `from '@paste/icons/cjs/coolIcon.js';`), the resolver will check the `package.json` file for which file to look up:

![image](https://user-images.githubusercontent.com/1592327/73694634-1471c980-469e-11ea-8bd2-9ac89290b7b2.png)

Since we're importing from the index file, only the exports from that file are available.  This is a neat trick because you package can have a bunch of different files with exports, but none get exposed to the user unless you explicitly export them from the entry point.  This allows us to test internal functions easily.

Reply:
Nice, thanks for the explanation